docs/components/marker.md

```diff
@@ -35,11 +35,35 @@ Offset of the marker from the left in pixels, negative number indicates left.
 ##### `offsetTop` {Number} - default: `0`
 Offset of the marker from the top in pixels, negative number indicates up.
 
+##### `draggable` {Boolean} - default `false`
+Allows this marker component to be dragged around the map. (Use `onDragEnd` to capture the final position and update `longitude` and `latitude`).
+
+##### `onDragStart` {Function}
+Called when a draggable marker starts being dragged.
+
+Parameters
+- `event` - The pointer event.
+  + `event.lngLat` - The geo coordinates where the drag started, as `[lng, lat]`.
+
+##### `onDrag` {Function}
+Continuously called while a draggable marker is being dragged.
+
+Parameters
+- `event` - The pointer event.
+  + `event.lngLat` - The geo coordinates of the drag event, as `[lng, lat]`.
+
+##### `onDragEnd` {Function}
```

Comment:
Is there a concept of DragCancel? (ESCAPE key etc)

Reply:
So technically there is a `pancancel` event in hammer which could theoretically warrant an onDragCancel call, but I couldn't figure out how to trigger it (dragging outside viewport, etc didn't do anything).

We could add support for escape key, though that would require another explicit keyup handler. If we're mostly trying to stay aligned with [mapbox draggable markers](https://www.mapbox.com/mapbox-gl-js/example/drag-a-marker/), they currently don't have a cancel event so it might be fine to omit for now.